hrms/payroll/doctype/salary_structure/salary_structure.json

```diff
@@ -239,44 +242,13 @@
  "idx": 1,
  "is_submittable": 1,
  "links": [],
- "modified": "2024-11-21 18:02:55.785057",
+ "modified": "2024-11-22 11:17:28.384386",
  "modified_by": "Administrator",
  "module": "Payroll",
  "name": "Salary Structure",
  "naming_rule": "Set by user",
  "owner": "Administrator",
- "permissions": [
```

Comment:
Default perms got removed accidentally IG

Reply:
they're removed while saving the updated doctype settings, couldn't figure out why, added them back